virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -0,0 +1,108 @@
+[id="virt-learn-more-about-openshift-virtualization"]
+= Learn more about deploying and managing virtual machines in your {product-title} cluster.
+include::modules/virt-document-attributes.adoc[]
+:context: virt-learn-more-about-openshift-virtualization
+
+toc::[]
+
+Use the following tables to find content to help you learn about and use {VirtProductName}.
+
+[id="cluster-administrator"]
+== Cluster Administrator
```

Comment:
Lowercase "Administrator" for sentence case?

```
== Cluster administrator
```

Reply:
Done